### DISCUSSION TITLE
Vertical swiper slider with 2 columns 2 slides per view

### DISCUSSION THREAD

**Discussion by vitalypushkov** [nolimits4web/swiper#5658] [General]
Guys, need help! i`ve benn searching for answer multi days but didnt find anything.
For me its not possible to make vertical swiper slider with 2 slides per view or 1 slide in each column.
i`ve been testing alot of option with js and css of slider but cannot reach result..

Maybe someone has done such project and has suggestions how to solve this trustrating problem.
![image](https://user-images.githubusercontent.com/102243228/164907276-03d12591-e382-4722-a292-5cd860e5a916.png)
my images are going right,not down even there is set to 2 columns.
Display flex with wrap either doesnt help with that.

Html code - 
![image](https://user-images.githubusercontent.com/102243228/164908821-2e4230fd-b7c5-4e3d-9e96-8dd27721f61d.png)
JS
![image](https://user-images.githubusercontent.com/102243228/164908911-7eef85d7-0b68-430d-aedb-34620a80e0d7.png)

css
![image](https://user-images.githubusercontent.com/102243228/164908921-97c7538d-593b-40c6-b5ca-8138d86674b7.png)

**Comment by s-andrzejewski** [comment]
@vitalypushkov Did you resolved it? I have the same problem.
